### PR TITLE
#158174589 feature(profile-reports) configure periodic reports

### PR DESCRIPTION
**Pull request by matthewacha**
#### What does this PR do?
- Add report frequency in Profile models class
- Apply migration

#### Description of Task to be completed?
Implementing the following changes to the files
- models.py
  - Adding report_frequency column in profile model
  - Adding logic to check which report frequency is stored

- accounts/views.py
  - Adding assignment for report_frequency form

- Run migrations

#### How should this be manually tested?
After cloning the repo, run the following commands
* make
* python manage.py test
* python manage.py makemigrations
* python manage.py migrate

#### Any background context you want to provide?
This will enable the developer to implement report frequency reminders; daily, monthly and weekly
in the user interface

##### Form for configuring periodic reports
screen shot 2018-07-02 at 11 11 08

##### Reports Dashboard
screen shot 2018-06-29 at 08 13 03

#### What are the relevant pivotal tracker stories?
[#158174589](https://www.pivotaltracker.com/n/projects/2177206)

### CODE REVIEW

**Comment by EdwinKato** [comment]
@Jpkat92 @matthewacha Resolve conflicts

**Comment by EdwinKato** [comment]
The UI only allows you to configure `daily`, `weekly` or `monthly` reports when you select the checkbox button above `Choose interval to send reports`. Is there a reason selecting an interval is tied to the monthly reports checkbox?

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
     - pip install -r requirements.txt
     - pip install braintree coveralls mock mysqlclient
 env:
-  - DJANGO=1.9
   - DB=postgres
 addons:
   postgresql: "9.4"
 script:
   - coverage run --omit=*/tests/* --source=hc manage.py test
-notifications:
```
Comment (EdwinKato) [inline .travis.yml]: Any particular reason this file was changed in this PR?

File: hc/accounts/tests/test_profile.py [hunk 2]
```diff
@@ -148,3 +148,39 @@ def test_it_updates_reports(self):
         response = self.client.post("/accounts/profile/", form)
         self.assertIn(b"Your settings have been updated!",response.content)
 
+    # Test configuring reports for daily, weekly and monthly durations
+    def test_configure_daily_reports(self):
+        url = "/accounts/profile/"
```
Comment (EdwinKato) [inline hc/accounts/tests/test_profile.py:153]: Use reverse for urls

**Review by EdwinKato — CHANGES_REQUESTED** [review]

**Comment by JaymesKat** [comment]
Hello Edwin, I have corrected the text on the UI for the configure report feature

**Comment by EdwinKato** [comment]
Nice, have a look at the other comments as-well

File: .travis.yml (hunk 1 above)
Comment (matthewacha) [reply .travis.yml]: The yml syntax was having a problem, which caused our pushes not to start a build so we reverted to not using on_failure. So I deleted and put it at the bottom. Solved the problem. I think it must have been the indentations

**Comment by EdwinKato** [comment]
@matthewacha resolve conflicts

**Review by EdwinKato — APPROVED** [review]

**Pull request merged**